src/components/CodeEditor/index.jsx

```diff
@@ -18,7 +18,10 @@ import './json-lint';
     width: '100%',
   },
 })
-/** Render an editor */
+/** Render an editor.
+ *
+ * * _Note: [material-ui](https://material-ui.com/) is a peer-dependency_
```

Comment:
No need to add an extra "*".

```
* _Note: [material-ui](https://material-ui.com/) is a peer-dependency_
```

A similar change is required for the others.